validation.md

```diff
@@ -1240,6 +1241,8 @@ The `filter` validator, which uses PHP's `filter_var` function, ships with Larav
 > [!WARNING]  
 > The `dns` and `spoof` validators require the PHP `intl` extension.
 
+Since email validation can be quite complex, you may use (Rule::email(#validating-emails) to fluently construct the rule.
```

Comment:
It looks like you have one parenthesis too many here:
```suggestion
Since email validation can be quite complex, you may use Rule::email(#validating-emails) to fluently construct the rule.
```